daemon/cluster/noderunner.go

```diff
@@ -271,7 +271,13 @@ func (n *nodeRunner) enableReconnectWatcher() {
 		if n.stopping {
 			return
 		}
-		config.RemoteAddr = n.cluster.getRemoteAddress()
+		remotes := n.cluster.getRemoteAddressList()
+		if len(remotes) > 0 {
+			config.RemoteAddr = remotes[0]
+		} else {
+			config.RemoteAddr = ""
+		}
```

Comment:
@aaronlehmann Remote addresses can change while the node is running. In the case of reconnect it should pick a new active one instead of reusing the previous remote that might be down.